composer.json

```diff
@@ -81,6 +83,7 @@
         "phpstan/phpstan-symfony": "^0.12.10",
         "psalm/plugin-symfony": "^2.0",
         "psr/event-dispatcher": "^1.0",
+        "sensio/framework-extra-bundle": "^5.6 || ^6.1",
```

Comment:
Do we need to support 2 versions?

Reply:
I used first only `6.1`, but @VincentLanglet suggested to support more https://github.com/sonata-project/SonataAdminBundle/pull/6950#pullrequestreview-617961191

Reply:
At least for master we should remove ^5 support

Reply:
I'm not against the removal of the `^5` support. 
My review was more about the conflict https://github.com/sonata-project/SonataAdminBundle/pull/6950#pullrequestreview-617961191

Reply:
Oh I see, but it is nice conflict matches require dev, in order to make sure our code works with the installed version of the user project

Reply:
Can we use only 6.1 then? @VincentLanglet and @franmomu ?

Reply:
Sure we can remove the test of 5.6 if you want, but the conflict should stay on 5.6 since it works with upper version

Reply:
Then dont change anything it does not make sense to conflict with different thing that you are installing :/

Reply:
> it is nice conflict matches require dev,

Conflict are here to say `This cannot works for sure with these versions`.
The feature required for this PR was introduced in the version 5.6.

Having test on all the possible version is another thing.

Reply:
If we keep `5.6` in the `conflict` section, to me it doesn't make sense to not test it as well, let's keep it this way and remove it after merging?